pkg/gui/controllers/helpers/search_helper.go

```diff
@@ -167,6 +169,31 @@ func (self *SearchHelper) CancelPrompt() error {
 	return self.c.PopContext()
 }
 
+func (self *SearchHelper) ScrollHistory(scrollIncrement int) error {
```

Comment:
Is a return value needed if all paths lead to `nil`?

Reply:
You're right. I removed the `return nil` statements in this fixup.
2a3bd0991680f3bac8deb9b000dd143af5dd0722